.circleci/config.yml

```
@@ -77,8 +77,9 @@ jobs:
             pip install cython
             pip install numpy==1.23.5
             pip install -r doc_requirements.txt
-            # `asv` pin because of slowdowns reported in gh-15568
-            pip install mpmath gmpy2 "asv==0.4.2" pythran ninja meson click rich-click doit pydevtool pooch
+            pip install mpmath gmpy2 \
+            "git+https://github.com/airspeed-velocity/asv@d4c868e52943894688e2db6abbf1ba2ab179cb84" \
+            pythran ninja meson click rich-click doit pydevtool pooch
```

With "extra line", I meant
```suggestion
            pip install mpmath gmpy2 pythran ninja meson click rich-click doit pydevtool pooch
            pip install "git+https://github.com/airspeed-velocity/asv@d4c868e52943894688e2db6abbf1ba2ab179cb84"
```